KS.Fiks.IO.Arkiv.Client/ForenkletArkivering/ArkivmeldingFactory.cs

```diff
@@ -458,13 +464,19 @@ public static Arkivmelding GetArkivmelding(ArkivmeldingForenkletNotat input)
                     {
                         dokumentbeskrivelse.Skjerming = new Skjerming()
                         {
-                            //skjermingshjemmel = input.nyttNotat.skjerming?.skjermingshjemmel,
-                            SkjermingDokument = "Hele"
+                            SkjermingDokument = SkjermingDokumentHele
                         };
                     }
+                    
+                    var filnavn = input.nyttNotat.hoveddokument.filnavn;
+                    
                     var dokumentobjekt = new Dokumentobjekt
                     {
-                        ReferanseDokumentfil = input.nyttNotat.hoveddokument.filnavn
+                        Format = filnavn.Substring(filnavn.LastIndexOf('.')),
```

Comment:
`filnavn.Substring(filnavn.LastIndexOf('.'))` 

Denne går litt igjen. Kunne vurdert å lage extension method på string eller en enkel statisk metode for å hente ut filending

Reply:
Enig. Jeg fikser